platform/ethereum/blockbook/transaction_test.go

```diff
@@ -182,6 +217,23 @@ func TestNormalizePage(t *testing.T) {
 						"from": "0x7d8bf18C7cE84b3E175b339c4Ca93aEd1dD166F1",
 						"to": "0x7d8bf18C7cE84b3E175b339c4Ca93aEd1dD166F1"
 					}
+				  },{
+					"id": "0xfea3444def47c77c5ede0f705d1bc84bb258d51fa454971985c181d6bc679144",
+					"coin": 60,
+					"from": "0x7d8bf18C7cE84b3E175b339c4Ca93aEd1dD166F1",
+					"to": "0x47331175b23C2f067204B506CA1501c26731C990",
+					"fee": "0",
+					"date": 1589249442,
+					"block": 0,
+					"status": "pending",
+					"sequence": 0,
+					"type": "contract_call",
```

Comment:
why does it says contract call?

Reply:
this test case is correct now, data: `0xdeadbeef`